src/components/Form/Stepper/Stepper.scss

```diff
@@ -0,0 +1,50 @@
+@import "../../../styles/main.scss";
+
+.stepperContainer {
+  display: flex;
+  padding: $size-xxxs;
+  border: 1px solid $greyscale-80;
+  position: relative;
+  margin-top: $size-l;
+
+  .decrease,
+  .increase {
+    border: 0;
+    width: 100%;
+    max-width: 50px;
+    height: 50px;
```

Comment:
we should probably use `rem` for `max-width` and `height`

Reply:
WIll do!